tasks/components/SubtaskView.tsx

```diff
@@ -51,19 +52,17 @@ export const SubtaskView = ({
   language,
   subtasks,
   taskID,
-  onChange
+  taskTemplateId,
+  onChange,
 }: PropsWithLanguage<SubtaskViewTranslation, SubtaskViewProps>) => {
+  const context = useContext(TaskTemplateContext)
+
   const translation = useTranslation(language, defaultSubtaskViewTranslation)
   const scrollableRef = useRef<SimpleBarCore>(null)
   const [scrollToBottomFlag, setScrollToBottom] = useState(false)
 
-  const isCreatingTask = !!taskID
-  const addSubtaskMutation = useSubTaskAddMutation(() => undefined, taskID)
-  const updateSubtaskMutation = useSubTaskUpdateMutation()
-  const deleteSubtaskMutation = useSubTaskDeleteMutation()
-  const subtaskToToDoMutation = useSubTaskToToDoMutation()
-  const subtaskToDoneMutation = useSubTaskToDoneMutation()
-
+  // TODO: Remove ?? '' once the mutate functions are passed properly
+  useSubTaskTemplateAddMutation(() => undefined, taskTemplateId ?? '')
```

Comment:
As you are using `noop` as default parameter, just leave it out.
```suggestion
  useSubTaskTemplateAddMutation(taskTemplateId ?? '')
```